.gitignore

```diff
@@ -31,3 +31,4 @@ build_elf_official.sh
 build_win32_official.cmd
 
 assets/
+.vscode/*
```

Comment:
This belongs in your own gitignore, not the project's. Besides, this folder will be used for https://github.com/Raid-Gaming/CoD4-Unleashed-Server/issues/34 so shouldn't be ignored either way.

Reply:
```suggestion

```